doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,158 @@
+.. _ug_zigbee_configuring_zboss_traces:
+
+Configuring ZBOSS traces in |NCS|
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The :ref:`nrfxlib:zboss` (ZBOSS stack) comes included in the |NCS| in a set of precompiled libraries, which can complicate the debugging process.
+To help with that, the ZBOSS stack can be configured to print trace logs that allow you to trace the stack behavior.
+This page describes how to enable and configure ZBOSS trace logs.
+
+Trace logs are printed in binary form and require access to stack source files, which are to be decoded into log messages.
+Enabling trace logs can help with the debugging process even if you cannot decode them (due to lack of access to stack source files).''
```

Comment:
Maybe we can change this to positive language and say that Nordic/Devzone Support team can decode them?

Reply:
That sounds like a good idea! I'm just curious what others think? (@greg-fer) and if there are other docs that have done this before? I can check in the meantime.